ox_bqpipeline/bqpipeline.py

```diff
@@ -324,8 +440,13 @@ def run_query(self, path, batch=False, wait=True, create=True,
         template = self.jinja2.from_string(template_str)
         query = template.render(**kwargs)
         client = self.get_client()
+        job_config = self.create_job_config(batch, dest, create, overwrite)
+        query_parameters = self.set_query_params(query_params)
```

Comment:
On second thought, `query_params` should be added as an arg to `create_job_config` and this logic moved there.

Reply:
Done.